docs/reference/mapping/params/copy-to.asciidoc

```diff
@@ -68,12 +68,58 @@ Some important points:
 `field_1` to `field_2` and `copy_to` on `field_2` to `field_3` expecting 
 indexing into `field_1` will eventuate in `field_3`, instead use copy_to 
 directly to multiple fields from the originating field.
```

Comment:
while you're here, this sentence makes zero sense

Reply:
rejigged